frontend/src/SelectLanguage.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import LOCALIZE from "./text_resources";
+import { Helmet } from "react-helmet";
+import { Button } from "react-bootstrap";
+import psc_logo from "./images/psc_logo.png";
+import canada_logo from "./images/canada_logo.png";
+import { setLanguage, LANGUAGES } from "./modules/LocalizeRedux";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+
+const styles = {
+  logo: {
+    padding: "10px 20px"
+  },
+  selectorContainer: {
+    width: 600,
+    height: 300,
+    backgroundColor: "#FFFFFF",
+    padding: 20,
+    border: "1px solid #00565e",
+    borderRadius: 5,
+    position: "absolute",
+    top: 0,
+    bottom: 0,
+    left: 0,
+    right: 0,
+    margin: "auto"
+  },
+  imageContainer: {
+    margin: "10px 0px"
+  },
+  buttonsContainer: {
+    margin: "10px 0px",
+    borderTop: "1px solid #CECECE"
+  },
+  langButton: {
+    margin: 10
+  }
+};
+
+class SelectLanguage extends Component {
+  static propTypes = {
+    // Props from Redux
+    setLanguage: PropTypes.func.isRequired
+  };
+
+  setLanguageToEnglish = () => {
+    this.props.setLanguage(LANGUAGES.english);
+  };
+
+  setLanguageToFrench = () => {
+    this.props.setLanguage(LANGUAGES.french);
+  };
+
+  render() {
+    return (
+      <div className="app">
+        <Helmet>
+          <title>{LOCALIZE.titles.CAT}</title>
+        </Helmet>
+        <div>
+          <div style={styles.selectorContainer}>
+            <div style={styles.imageContainer}>
+              <img
+                alt={LOCALIZE.mainTabs.psc}
+                src={psc_logo}
+                width="370"
+                className="d-inline-block align-top"
+                style={styles.logo}
+              />
+              <img
+                alt={LOCALIZE.mainTabs.canada}
+                src={canada_logo}
+                width="120"
+                className="d-inline-block align-top"
+                style={styles.logo}
+              />
+            </div>
+            <h2>Competency Assessment Tool</h2>
+            <h2 lang={LANGUAGES.french}>Outil d'évaluation des compétences</h2>
+            <div style={styles.buttonsContainer}>
```

Comment:
Shouldn't we put these in LOCALIZE, even if these don't have translation, to have all the app wording at the same place?

Reply:
I was debating that myself, to be honest, but you'd have a copy in en and fr, and maintain both changes... so like the idea of a centralized location, but don't think Localize is the right spot... idk. Willing to discuss, though

Reply:
These are not in LOCALIZE because they will always be shown in both languages like this. We do the same with the `Translation` button.

Reply:
Then, we should maybe create a file and put all the wording of this type at the same place? We can do that a another PR though..

Reply:
I'm not sure I understand the reasoning here - let's discuss in person and do a followup PR, since it follows our current standard.

Reply:
It is just an idea in order to have all the wording at the same place. I'm thinking more for the future maintenance of this application. I would be easier for sure if all the wording is at the same place.
But it's fine like that for now..